app/diagnostics.go

```diff
@@ -138,15 +136,14 @@ func (a *App) trackActivity() {
 	var incomingWebhooksCount int64
 	var outgoingWebhooksCount int64
 
-	dailyActiveChan := a.Srv.Store.User().AnalyticsActiveCount(DAY_MILLISECONDS)
```

Comment:
Same here, we expect to use the "explicit async" approach.

Reply:
Here, still expecting to migrate it to "explicit async" approach.

Reply:
@jespino since currently there's no `explicit async` approach in this file.
I try make it as similar as with `app/diagnostic` approach. I just still need to print error if got it, so error will not just gone.